### DISCUSSION TITLE
Welcome to GPU-Viewer Discussions!

### DISCUSSION THREAD

**Discussion by arunsivaramanneo** [arunsivaramanneo/GPU-Viewer#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by unexploredtest** [comment]
First, I wanna thank for for providing such a useful tool, especially the fact that it supports most graphics API. Much appreciated.
One nice thing would be an addition of Dark Mode, light mode makes my eyes (and many others) sore
Also, sometimes it GPU-Viewer can be slow, not so important but can be annoying sometimes

**Reply by arunsivaramanneo** [reply]
Thank you So much .. i understand about the dark mode and it was in my to do list but unfortunately i am not getting time in supporting/developing this application some time now due to my busy schedule and with my health... once i  get time .. this will be the first thing i want to deliver... thanks again...

**Reply by unexploredtest** [reply]
I see, take your time. Thanks and take care!
